examples/listbox/js/listbox.js

```diff
@@ -196,6 +218,14 @@ aria.Listbox.prototype.checkKeyPress = function (evt) {
         this.focusItem(nextUnselected);
       }
       break;
+    case 65:
+      // handle control + A
+      if (this.multiselectable && evt.ctrlKey) {
```

Comment:
To answer @zcorpan 's question:
> So on macOS, the usual shortcut for select all is Command+A, not Control+A. How should we handle this?

Detect macOS and change the shortcut from Control+A to Command+A

Something like this should do it:
```suggestion
      var isMac = window.navigator.platform.indexOf("Mac") !== -1;
      var mod = isMac ? evt.metaKey : evt.ctrlKey;
      if (this.multiselectable && mod) {
```

Reply:
I don't think we need to detect Mac. I just added `(evt.ctrlKey || evt.metaKey)`. That should work for Mac, and doesn't cause issues with the Windows key, since Windows + A is eaten by the OS anyway.

@zcorpan let me know if it's working for you with that change

Reply:
> Windows + A is eaten by the OS anyway.

Ha! I did not know that. :)
Seems to open up the notification sidebar.
Whaddaya know.
Thanks for the lesson!

Reply:
What will this logic end up doing on Linux where I believe we want it to be ctrl-a? There could be other platforms, even some mobile with BT keyboards that will allow these commands to be used. I just pushed a change to the keyboard documentation, but I don't believe what I wrote is entirely accurate using this logic.

Reply:
Hmmm... now that I'm looking at the logic, I believe it makes both `Ctrl+A` and `Cmd+A` work on macOS. Do we want that?

Reply:
@mcking65 
I don't think there's a [KeyboardEvent.metaKey](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey) on Linux. Hopefully someone can confirm.

The doc says the only other `metaKey` is the Windows key on Windows keyboards, and @smhigley mentioned that Windows + A is eaten by the OS.

The code within the case for the 'A' (65) key being pressed is:
`if (this.multiselectable && (evt.ctrlKey || evt.metaKey))`

so the logic would work as follows on Linux:
"if the list is multiselectable and the control key was pressed (with the 'A')"

So the code `|| evt.metaKey` would never be evaluated.

So I think the logic is ok for Linux, but I don't think it's ok for macOS, unless we want both ctrl+A and command-A to select all.

Reply:
@mcking65 
You are correct about the keyboards! I googled a bit and discovered that some older Linux keyboards and even some recent bluetooth android keyboards have a meta key (apparently for the mobile ones, the "search" key maps to metaKey). So even though those keyboards are likely rare, we don't want meta+A to select all. We only want that on Mac.

@smhigley I think we need to go with the code that detects Mac:
```
var isMac = window.navigator.platform.indexOf("Mac") !== -1;
var mod = isMac ? evt.metaKey : evt.ctrlKey;
if (this.multiselectable && mod) {
```